### DISCUSSION TITLE
Millions of tar files in /var/lib/teleport/log/records/

### DISCUSSION THREAD

**Discussion by git-lew** [gravitational/teleport#9149] [Q&A]
#was not able to log in. SSH into the server and saw 100% usage on /dev/sda1 using df -h
#teleport is generating an impressive amount of records, related to inode
#running the following shows 100% usage
df -i 

#running the following, I was able to identify the path as the main location for approx 2 million inode entries
for i in /*; do echo $i; find $i |wc -l; done

#adding the largest entry in each run, I eventually reached
for I in /var/lib/teleport/log/records/*; do echo $i; find $i |wc -l; done

#Has anyone else seen this?

**Comment by darius-fritsch** [comment]
I would recommend checking your logs. There may be some problems uploading these records.
